### DISCUSSION TITLE
Ranch error: sequence without chain ID found

### DISCUSSION THREAD

**Discussion by Lisa-Casteller** [biosaxs-com/atsas-community#1929] [Mixtures and flexible systems] [answered]
Hi all,

I'm having issues setting up ranch. I keep getting this error:

 error: assignment.txt: sequence without chain ID found, must be single letter A-Z

Using the following input:

ranch --repetitions 10000 --prefix pool/pep_ sequence.fasta assignment.txt

and input files:

**cat assignment.txt** 
A 1 551 disordered

**sequence.fasta**
"> A
ALPSEEEFAERKRPEQPALATFAMPDVPPAPTPAEPAAPVVAPAPKAAPATPAAPAQPGLLSRFFGALKALFSGGEETKPTEQPAPKAEAKPERQQDRRKPRQNNRRDRNERRDTRSERTEGSDNREENRRNRRQAQQQTAETRESRQQAEVTEKARTADEQQAPRRERSRRRNDDKRQAQQEAKALNVEEQSVQETEQEERVRPVQPRRKQRQLNQKVRYEQSVAEEAVVAPVVEETVAAEPIVQEAPAPRTELVKVPLPVVAQTAPEQQEENNADNRDNGGMPRRSRRSPRHLRVSGQRRRRYRDERYPTQSPMPLTVACASPELASGKVWIRYPIVRPQDVQVEEQREQEEVHVQPMVTEVPVAAAIEPVVSAPVVEEVAGVVEAPVQVAEPQPEVVETTHPEVIAAAVTEQPQVITESDVAVAQEVAEQAEPVVEPQEETADIEEVVETAEVVVAEPEVVAQPAAPVVAEVAAEVETVAAVEPEVTVEHNHATAPMTRAPAPEYVPEAPRHSDWQRPTFAFEGKGAAGGHTATHHASAAPARPQPVE"

I have used 'od -c' and 'echo -n "A 1 551 disordered" > assignment.txt' to make sure there weren't any hidden characters, but no matter what I do, the A does not seem to be recognised.

How should I proceed? 

Thank you,
all

**Comment by maxim-biosaxs** [comment, marked as answer]
Please try to run the application in this way:
ranch --repetitions 10000 --prefix pool/pep_  assignment.txt sequence.fasta

**Reply by Lisa-Casteller** [reply]
Thank you, it works now!
